exercises/binary-search/.meta/template.j2

```diff
@@ -0,0 +1,27 @@
+{%- import "generator_macros.j2" as macros with context -%}
+
+{%- macro test_call(case) %}
+            {{ case["property"] }}(
+                {% for arg in case["input"].values() -%}
+                "{{ arg }}",
+                {% endfor %}
```

Comment:
As we don't have any guarantee that the order of these arguments won't change in the canonical-data, might I suggest:
```
{{ case["property"] }}(
    case["input"]["array"],
    case["input"]["value"]
)
```

Reply:
The other issue with the current changes is that neither the `array` input nor the `value` input are a string, but the for loop there treats them as such.

Reply:
Changes made as suggested. Thanks.